src/components/section/Deck.js

```diff
@@ -30,16 +30,19 @@ const useStyles = makeStyles((theme) => ({
       height: "40vw",
     },
   },
-
 }));
 
 const Deck = ({ elements }) => {
   const theme = useTheme();
   const classes = useStyles();
   const universalClasses = universalStyles();
   let content = elements[0];
-  const title = !!content.title ? content.title : content.name;
-  const author = !!content.author ? content.author.name : "";
+  let title = "";
+  let author = "";
+  if (content) {
+    author = !!content.author ? content.author.name : "";
+    title = !!content.title ? content.title : content.name;
+  }
```

Comment:
Try if this works (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):

```suggestion
  const title = content?.title ?? content?.name ?? '';
  const author = content?.author?.name ?? '';
```